tools/enso4igv/IGV.md

```diff
@@ -136,6 +136,12 @@ icon in the _Stack View_ to get from graph node to source. Select a drop down
 widget in the editor toolbar to show you what compiler nodes as associated with
 currently selected line.
 
+## Syntax Coloring
+
+To enable syntax coloring you may need to download and install text mate
+module into IGV. Download [its NBM file](https://repo1.maven.org/maven2/org/netbeans/api/org-netbeans-modules-textmate-lexer/RELEASE260/org-netbeans-modules-textmate-lexer-RELEASE260.nbm)
+and install in _Plugins_ dialog.
```

Comment:
We are having problems with dependency on TextMate grammer support module: https://lists.apache.org/thread/h3h0r89d6co4r0o3x0ybzscjstf4ynzw

Not sure what's the best resolution yet...